internal/gossiper/proposer.go

```diff
@@ -193,7 +193,7 @@ func (g *Proposer) HandleAppGossip(ctx context.Context, nodeID ids.NodeID, msg [
 	var seen int
 	for _, tx := range txs {
 		// Verify signature async
-		txDigest, err := tx.Digest()
+		txDigest, err := tx.Preimage()
```

Comment:
can we update the local variable names where we're updating the function name so that they match and we get rid of the misnomer? Same comment throughout.

Reply:
done